src/elements/table/table-wrapper/table-wrapper.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { assert } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+
+import { fixture } from '../../core/testing/private.js';
+
+import { SbbTableWrapperElement } from './table-wrapper.js';
+
+describe(`sbb-table-wrapper with ${fixture.name}`, () => {
+  let element: SbbTableWrapperElement;
+
+  beforeEach(async () => {
+    element = await fixture(html`<sbb-table-wrapper></sbb-table-wrapper>`, {
+      modules: ['./table-wrapper.ts'],
+    });
```

Comment:
```suggestion
    element = await fixture(html`<sbb-table-wrapper></sbb-table-wrapper>`);
```

Reply:
To be sure, we need to explicitly declare modules with the SSR fixture, correct?

Reply:
yes but only for ssr in the ssr file. in all other spec files we don't need it.